test/chain/TransactionBuilder.js

```diff
@@ -420,7 +420,7 @@ describe("TransactionBuilder", () => {
                 whitelist_markets: [],
                 blacklist_markets: [],
                 description: JSON.stringify({main: "", market: ""}),
-                extensions: null
+                extensions: {}
```

Comment:
Why is this change necessary?

Reply:
yes, null causes `null pointer exceptions` when trying to resolve some field of extension on object which doesn't contain such extension